### PR TITLE
Add CSRF Protection

### PR DESCRIPTION
**Pull request by hepsimo**
Resolves:
SEC-87
SEC-88
SEC-89

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: test/routers/confirmCompany.int.ts [hunk 1]
```diff
@@ -14,6 +14,10 @@ mockGetCompanyProfile.mockResolvedValue(validCompanyProfile);
 
 describe("ConfirmCompany router/handler integration tests", () => {
 
+    beforeAll(() => {
+        // middlewareMocks.mockCsrfProtectionMiddleware.mockClear();
```
Comment (mwejuli-ch) [inline test/routers/confirmCompany.int.ts:18]: comment can be removed, and by extension, the `beforeAll` block (if there's no code in it)

**Review by mwejuli-ch — CHANGES_REQUESTED** [review]

File: test/routers/confirmCompany.int.ts (hunk 1 above)
Comment (hepsimo) [reply test/routers/confirmCompany.int.ts:18]: Thanks, i'll fix this

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by mwejuli-ch — APPROVED** [review]

**Review by PhilipGlover — APPROVED** [review]
- code follows CSRF confluence guide
- have completed the Individual without changing the `_csrf` token
- have changed the token on each page and CSRF error screen from `web-security-node` library displayed
- tests pass
```
Test Suites: 1 skipped, 32 passed, 32 of 33 total
Tests:       2 skipped, 108 passed, 110 total
Snapshots:   0 total
Time:        5.084 s, estimated 7 s
Ran all test suites.
```

**Pull request merged**